scipy/signal/ltisys.py

```diff
@@ -1024,37 +1490,168 @@ def to_tf(self, **kwargs):
 
         """
         return TransferFunction(*ss2tf(self._A, self._B, self._C, self._D,
-                                       **kwargs))
+                                       **kwargs), **self._dt_dict)
 
     def to_zpk(self, **kwargs):
         """
-        Convert system representation to `ZerosPolesGain`.
+        Convert system representation to `ZerosPolesGain`.
+
+        Parameters
+        ----------
+        kwargs : dict, optional
+            Additional keywords passed to `ss2zpk`
+
+        Returns
+        -------
+        sys : instance of `ZerosPolesGain`
+            Zeros, poles, gain representation of the current system
+
+        """
+        return ZerosPolesGain(*ss2zpk(self._A, self._B, self._C, self._D,
+                                      **kwargs), **self._dt_dict)
+
+    def to_ss(self):
+        """
+        Return a copy of the current `StateSpace` system.
+
+        Returns
+        -------
+        sys : instance of `StateSpace`
+            The current system (copy)
+
+        """
+        return copy.deepcopy(self)
```

Comment:
Because the `lti` class behaves the same way. I think back when we added this the motivation was for `StateSpace(sys)` to work the same way as `sys.to_ss()`. In hindsight I probably would prefer not copying as well.

Notice that there are the `_as_ss()` functions as part of the class to avoid unecessary copying. These could be made part of the public API as well.